chips/stm32f303xc/src/wdt.rs

```diff
@@ -164,9 +160,13 @@ impl kernel::watchdog::WatchDog for WindoWdg {
 
     fn suspend(&self) {
         if self.enabled.get() {
-            unsafe {
-                WATCHDOG_SLEEP_FLAG = true;
-            }
+            self.clock.disable();
+        }
+    }
+
+    fn resume(&self) {
+        if self.enabled.get() {
+            self.clock.enable();
```

Comment:
`resume()` should probably also tickle the watchdog

Reply:
I don't think it is absolutely necessary to tickle it here. The watchdog value is not decremented during sleep because the clock is disabled right before it, in the suspend function. More so, the next thing that happens exactly after resume is actually the first tickle at the beginning of the loop block. I've got no problem also adding one here too, tho.

Reply:
oh, that makes sense, I think it is fine either way then.